proc.go

```diff
@@ -108,6 +126,8 @@ func newProc(w, h int) *Proc {
 	proc.cfg.th = material.NewTheme(gofont.Collection())
 	proc.stk.cur().stroke.style.Width = 2
 
+	proc.newWindow = appWindowFunc
```

Comment:
```suggestion
	proc.newWindow = appWindow
```
just default to the "normal" `func`.